extensions/sql-migration/src/api/utils.ts

```diff
@@ -1170,6 +1172,72 @@ export async function clearDropDown(dropDown: DropDownComponent): Promise<void>
 	await dropDown.updateProperty('values', []);
 }
 
+export async function getRecommendedConfiguration(targetType: MigrationTargetType, model: MigrationStateModel): Promise<string[]> {
+
+	if (!hasRecommendations(model)) {
+		return [];
+	}
+	let recommendation;
+	switch (targetType) {
+		case MigrationTargetType.SQLVM:
+			// elastic model currently doesn't support SQL VM, so show the baseline model results regardless of user preference
+			recommendation = model._skuRecommendationResults.recommendations?.sqlVmRecommendationResults[0];
+
+			// result returned but no SKU recommended
+			if (!recommendation?.targetSku) {
+				return [constants.SKU_RECOMMENDATION_NO_RECOMMENDATION];
+			}
+			else {
+				const vmConfiguration = constants.VM_CONFIGURATION(
+					recommendation.targetSku.virtualMachineSize!.sizeName,
```

Comment:
Hope here we are covering Premium SSD v2 new format of SKU summery and details.

Reply:
yes it is covered